hw03_NatsarS.py

```diff
@@ -0,0 +1,65 @@
+#################################################################################
+# Author: Salaam Natsar
+# Username: NatsarS
+#
+# Assignment: Homework 03
+# Purpose: Created a sun flower
+# Google Doc Link: https://docs.google.com/document/d/1gRRxdAQvTMhQ6pwa0BwnBMu09rZ2cglwKLnd5j2zvpg/edit?usp=sharing
+#################################################################################
+# Acknowledgements:
+# https://stackoverflow.com/questions/32804572/how-to-hide-a-turtle-icon-pointer-in-python
+#
+#################################################################################
+
+import turtle
+sunflower = turtle.Turtle()
+sunflower.speed(0)
+s=turtle.Screen()
+s.bgcolor('light green')
```

Comment:
There shouldn't be any code outside of a function. This probably belongs at the beginning of main (since it is the first code that is running, currently). 

To use sunflower inside your functions later, you need to pass it into the functions:

```
def petals(sunflower):
    ....

def main():
    sun = turtle.Turtle()
    petals(sun)
```